<launch>
  <!-- 你可以在 sim_gazebo_rviz_start.sh 第8步调用它 -->
  <node pkg="diff_pure_pursuit_local_planner"
        type="diff_pure_pursuit_node"
        name="diff_pure_pursuit_local_planner"
        output="screen">

    <!-- 对齐你当前链路 -->
    <param name="topic_path" value="/opt_path"/>
    <param name="topic_odom" value="/truth_pose_odom"/>
    <param name="topic_map" value="/local_map_esdf"/>


    <!-- 输出给底盘控制，你也可以 remap 到 /cmd_vel_auto -->
    <param name="topic_cmd"  value="/chassis_control"/>

    <remap from="/chassis_control" to="/cmd_vel_auto"/> 
    <!-- 可调参数 -->
    <param name="lookahead_dist" value="0.8"/>
    <param name="goal_tolerance" value="0.25"/>

    <param name="max_v" value="0.6"/>
    <param name="max_w" value="1.2"/>
    <param name="min_v" value="0.05"/>

    <param name="k_heading" value="1.8"/>
    <param name="v_scale" value="0.8"/>

    <param name="cmd_rate" value="30.0"/>
    <param name="timeout_path" value="1.0"/>
    <param name="timeout_odom" value="0.5"/>
  </node>
</launch>
